File: 4344.py
# 4344 py

n = int(input())

for _ in range(n):
    arr = list(map(int,input().split()))
    c = arr.pop(0)
    avg = sum(arr[:c]) / c
    result = 0
    for i in arr:
        if i > avg:
            result += 1
    result = result / c * 100
    print(f'{result:.3f}%')

File: 4344.c
// 4344 C

#include <stdio.h>

int    main(void)
{
	int		n;
	int		c;
	int		arr[1000];
	float	avg = 0;
	float	result = 0;

	scanf("%d", &n);
	for (int i = 0; i < n; i++)
	{
		scanf("%d", &c);
		for (int j = 0; j < c; j++)
		{
			scanf("%d", &arr[j]);
			avg += arr[j];
		}
		avg /= c;
		for (int k = 0; k < c; k++)
		{
			if (arr[k] > avg)
				result++;
		}
		result = result / c * 100;
		printf("%.3f%%\n", result);
		result = 0;
		avg = 0;
	}
}